### PR TITLE
Register upgrade plan for musselnet-v2

### PR DESCRIPTION
**Pull request by orkunkl**
This change is required to succesfully executing a cosmovisor assisted upgrade. Reference: https://godoc.org/github.com/cosmos/cosmos-sdk/x/upgrade#hdr-Performing_Upgrades

### CODE REVIEW

**Comment by alpe** [comment]
@orkunkl I have modified the target branch as discussed. Please update the "upgrade name". Otherwise looks good 👍

**Review by alpe — APPROVED** [review]

File: app/app.go [hunk 1]
```diff
@@ -645,3 +645,9 @@ func initParamsKeeper(appCodec codec.BinaryMarshaler, legacyAmino *codec.LegacyA
 
 	return paramsKeeper
 }
+
+func (app *WasmApp) registerUpgradePlans() {
+	app.upgradeKeeper.SetUpgradeHandler("v0.13.1", func(ctx sdk.Context, plan upgradetypes.Plan) {
```
Comment (ethanfrey) [inline app/app.go:650]: Looks good. I would just name this "musselnet-v2" or something like that.

**Review by ethanfrey — APPROVED** [review]

**Comment by ethanfrey** [comment]
I heard some upgrade issues with -rc6 (maybe earlier). It would be good to do some local testing.

Once #354 is merged, please rebase on/merge from that one. And it would be good to locally test a (one node) upgrade from the current musselnet version (0.12.0?) to this commit.

I would be happy to report any issues to the cosmos sdk team. Or report that this worked without problems as a positive test.

**Comment by ethanfrey** [comment]
We will not perform this.
I will close the PR, but the branch is a good example of an upgrade script. Let's not delete the branch

**Pull request closed without merging**